partiql-ast/src/main/java/org/partiql/ast/ddl/CreateTable.java

```diff
@@ -0,0 +1,66 @@
+package org.partiql.ast.ddl;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.partiql.ast.AstNode;
+import org.partiql.ast.AstVisitor;
+import org.partiql.ast.IdentifierChain;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+@Builder(builderClassName = "Builder")
+@EqualsAndHashCode(callSuper = false)
+public class CreateTable extends Ddl {
+
+    @NotNull
+    public final IdentifierChain name;
+
+    @Nullable
```

Comment:
Should this be nullable? In your factory method, the `columns` argument is non-null.

Reply:
Good callout: 

In case of 
```
CREATE TABLE foo
```

do we prefer List<ColumnDefinition> and List<TableConstraint> constraints to be null, or do we prefer those to be emptyList()